test/policies/stripe_account_policy_test.exs

```diff
@@ -0,0 +1,146 @@
+defmodule CodeCorps.StripeAccountPolicyTest do
+  use CodeCorps.PolicyCase
+
+  import CodeCorps.StripeAccountPolicy, only: [index?: 2, show?: 2]
+  import CodeCorps.StripeAccount, only: [create_changeset: 2]
+
+  alias CodeCorps.StripeAccount
+
+  describe "index?" do
+    test "returns true when user is an admin" do
+      user = build(:user, admin: true)
+      organization = insert(:organization)
+
+      assert index?(user, %{organization_id: organization.id})
+    end
+
+    test "returns false when user is not member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      refute index?(user, %{organization_id: organization.id})
+    end
+
+    test "returns false when user is pending member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      insert(:organization_membership, role: "pending", member: user, organization: organization)
+
+      refute index?(user, %{organization_id: organization.id})
+    end
+
+    test "returns false when user is contributor of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      insert(:organization_membership, role: "contributor", member: user, organization: organization)
+
+      refute index?(user, %{organization_id: organization.id})
+    end
+  end
+
+  describe "show?" do
+    test "returns true when user is an admin" do
+      user = build(:user, admin: true)
+      organization = insert(:organization)
+
+      assert show?(user, organization)
+    end
+
+    test "returns false when user is not member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      refute show?(user, organization)
+    end
+
+    test "returns false when user is pending member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      insert(:organization_membership, role: "pending", member: user, organization: organization)
+
+      refute show?(user, organization)
+    end
+
+    test "returns false when user is contributor of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      insert(:organization_membership, role: "contributor", member: user, organization: organization)
+
+      refute show?(user, organization)
+    end
+
+    test "returns true when user is admin of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+
+      insert(:organization_membership, role: "admin", member: user, organization: organization)
+
+      assert show?(user, organization)
```

Comment:
I think we just want owner at the lowest level of privilege here.

I would also consider grouping your trues and your falses together.